mobile/lib/modules/backup/providers/manual_upload.provider.dart

```diff
@@ -274,7 +274,7 @@ class ManualUploadNotifier extends StateNotifier<ManualUploadState> {
     // The app is currently in background. Perform the necessary cleanups which
     // are on-hold for upload completion
     if (appState != AppStateEnum.active && appState != AppStateEnum.resumed) {
-      ref.read(appStateProvider.notifier).handleAppInactivity();
+      ref.read(backupProvider.notifier).cancelBackup();
```

Comment:
We used to have a guard in `handleAppInactivity` to stop all cleanup from executing when manual upload was in progress but to defer them to after the backup is completed and the app is still in the paused state. With this change, websockets that were not disconnected during app pause since manual upload was in progress, will not be disconnected even after the upload is completed. Can this be changed to call `handleAppPaused` instead?

Reply:
I've changed handleAppPaused to always close the websocket. So, only the manual backup can continue to run and needs to be cleaned up later.

 Or does the manual backup need the websocket?
I don't think so: when the app is resumed, the state is synced from the server and the websocket is connected again.